package.json

```diff
@@ -12,8 +12,9 @@
     "build": "ember build",
     "preversion": "git clean -df && npm test",
     "start": "ember server",
-    "test": "ember try:testall",
-    "docs": "node ./docs"
+    "test": "ember try:testall && mocha tests/**/*-test-node.js",
+    "docs": "node ./docs",
+    "mocha": "mocha tests/**/*-test-node.js"
```

Comment:
We don't need node tests for this feature now that we don't have the `enabledInProd` option